### PR TITLE
more rustflags opts

### PR DESCRIPTION
**Pull request by SashaXser**
Added the tune-cpu option as well as target-cpu for macos-arm64. I'll leave it as a draft for now

### CODE REVIEW

**Comment by gz83** [comment]
Have you done any performance comparison testing? Sometimes adding too many optimization parameters can be counterproductive.

In addition, some of the parameters you added may not be appropriate, and there are also wrong parameters.

File: mozconfigs/mozconfig [hunk 1]
```diff
@@ -98,10 +98,10 @@ elif test "$USE_PGO"; then
 fi
 
 # Compiler, Linker, and Rust flags
-export CFLAGS="-O3 -msse3 -mssse3 -msse4.1 -msse4.2 -mavx -maes"
-export CPPFLAGS="-O3 -msse3 -mssse3 -msse4.1 -msse4.2 -mavx -maes"
-export CXXFLAGS="-O3 -msse3 -mssse3 -msse4.1 -msse4.2 -mavx -maes"
+export CFLAGS="-O3 -msse3 -mssse3 -msse4.1 -msse4.2 -mavx -maes -mtune=sandybridge"
+export CPPFLAGS="-O3 -msse3 -mssse3 -msse4.1 -msse4.2 -mavx -mae -mtune=sandybridges"
```
Comment (gz83) [inline mozconfigs/mozconfig:102]: Nit: -mae?

**Review by gz83 — CHANGES_REQUESTED** [review]

**Comment by SashaXser** [comment]
> Have you done any performance comparison testing? Sometimes adding too many optimization parameters can be counterproductive.
> 
> In addition, some of the parameters you added may not be appropriate, and there are also wrong parameters.

I can't run the tests right now, can you run the tests for me?

**Comment by gz83** [comment]
Sorry, I don't have any resources right now to test the performance differences

**Comment by SashaXser** [comment]
If you don't like it, close it. I don't want to do anything else.

**Pull request closed without merging**